### DISCUSSION TITLE
Select-Scroll in List

### DISCUSSION THREAD

**Discussion by BeyondSmash** [ScanMountGoat/ssbh_editor#92] [Ideas]
CrossMod has a feature I enjoy when experimenting with entries. Select-scroll in list. Here's a video example that seems to be absent even in the latest SSBH:

https://user-images.githubusercontent.com/110957189/188065486-8dfd89c3-d1a3-4624-a8ce-5eb08c403ea0.mp4

**Comment by ScanMountGoat** [comment]
The UI library doesn't provide this by default. I'm not sure how difficult this will be to add. I'll try and look into it at some point.

**Reply by ScanMountGoat** [reply]
This feature is being tracked by issue https://github.com/ScanMountGoat/ssbh_editor/issues/101.
